### DISCUSSION TITLE
Auth 404'ed

### DISCUSSION THREAD

**Discussion by LitoStarr** [UltimaHoarder/UltimaScraper#1330] [Q&A]
Site: 0 = onlyfans | 
0
Choose Profile
Names: Username = username  | 0 = All | 1 = default
0
Auth (V1) Attempt 1/10
Auth 404'ed
Auth (V1) Attempt 2/10
Auth 404'ed
Auth (V1) Attempt 3/10
Auth 404'ed
Auth (V1) Attempt 4/10
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d79c40>
Auth 404'ed
Auth (V1) Attempt 5/10
Auth 404'ed
Auth (V1) Attempt 6/10
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d28730>
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d30b50>
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d47c70>
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d06c70>
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d47460>
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d28b50>
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d74460>
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d4ebe0>
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d4eee0>
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d81d30>
Auth 404'ed
Auth (V1) Attempt 7/10
Auth 404'ed
Auth (V1) Attempt 8/10
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x110d90cd0>
Auth 404'ed
Auth (V1) Attempt 9/10
Auth 404'ed
Auth (V1) Attempt 10/10
Auth 404'ed
Scraping Paid Content
Scraping Subscriptions
There's nothing to scrape.
Archive Completed in 0.15 Minutes
Pausing scraper for 0 seconds.
Site: 0 = onlyfans | 

Everything in my .profile auth is correctly input. I've deleted and reinstalled the program to no avail.

**Comment by kingknuts** [comment]
Happens the same to me... since I updated to latest commit, some of the latest python  requirements are not working. I’m trying to find what is causing that.

**Reply by LitoStarr** [reply]
It turns out that you're gonna have to download the previous version. The latest repository is a dud, in my opinion. Got some kinks it gotta iron.

**Reply by saka83838383** [reply]
May I know which version is working right now? It happens to me as well, struggling with 7.4.1, 404 all the way. Tried the 7.3, still not working.

**Reply by Tyemnata** [reply]
@saka83838383 Download the latest python, update as per pip install --upgrade --user -r requirements.txt in the read me (just in case), then run the updater as per: https://github.com/DIGITALCRIMINAL/OnlyFans/discussions/1696

I just set this up for the first time and have it running.
